scripts/OpdrachtThree/Opdracht-3.js

```diff
@@ -0,0 +1,32 @@
+// //Opdracht 3a
+// const jobChoice = prompt(`Je koos voor marketing. Over welke functie wil je meer weten?
+// Voer een getal tussen de 0 en 4 in.  0: ${departments.marketing[0].title}
+// 1: ${departments.marketing[1].title} 2: ${departments.marketing[2].title}
+// 3: ${departments.marketing[3].title} 4: ${departments.marketing[4].title}`)
+//
```

Comment:
${departments.marketing[0].title} moet ${departments.marketing.jobs[0].title} zijn, bij mij crashed het programma  hier op : Opdracht-3.js:3 Uncaught TypeError: Cannot read properties of undefined (reading 'title')